test/test_client.py

```diff
@@ -586,17 +597,17 @@ def test_no_args_no_kwargs(self):
 class TestBuildUrl(ClientTest):
 
     def test_build_url_positional(self):
-        expected = 'https://fake.taskcluster.net/v1/two_args_no_input/arg0/arg1'
+        expected = 'https://tc-tests.localhost/api/fake/v1/two_args_no_input/arg0/arg1'
```

Comment:
I really liked the self.apiPath pattern used in the previous tests, maybe it's applicable here too?